.gitignore

```diff
@@ -1,2 +1,3 @@
 # Put your Confluence username, API key, and domain in this.
-.env
\ No newline at end of file
+.env
+pubdoc
```

Comment:
Why's pubdoc in pubdoc's .gitignore?

Reply:
When you type `go build` it names the executable `pubdoc`. I don't want to commit binary machine code. 

I don't really do go often. Are you building some other way.

Reply:
Just skip the build and run directly with `go run main.go`

Reply:
I think for CI pipelines they typically have a directory in the repo reserved for build artifacts and put that directory in the .gitignore

Reply:
Let's not get ahead of ourselves. 
Regardless, I'm keeping the gitignore unless you put up a fight. Then I'll just give up.

Reply:
Hahaha, and yea that's fine. Maybe just a `#` comment to explain that `pubdoc` is intended to be the name of the binary if someone builds with `go build` ?

Reply:
Jack...
![courage-courage-cowardly-dog](https://github.com/user-attachments/assets/cb5eaa93-cf94-4985-a8b1-f496c8a2fa8c)